aiokatcp/server.py

```diff
@@ -498,8 +521,8 @@ def mass_inform(self, name: str, *args: Any) -> None:
             Fields for the inform
         """
         msg = core.Message.inform(name, *args)
-        for conn in self._connections:
-            conn.write_message(msg)
+        for conn in list(self._connections):
```

Comment:
On line 420 there is a comment noting why a copy is made.  Suggest something similar here:
`# Copy the connection list, to avoid mutation while iterating`

Reply:
Good idea. I've added a comment along those lines.